sensor/common/networkflow/manager/manager_impl.go

```diff
@@ -531,14 +539,17 @@ func (m *networkFlowManager) enrichProcessesListening(hostConns *hostConnections
 
 	prevSize := len(hostConns.endpoints)
 	for ep, status := range hostConns.endpoints {
-		if ep.processKey == nil {
+		if ep.processKey.processArgs == "" &&
+			ep.processKey.processName == "" &&
+			ep.processKey.processExec == "" {
```

Comment:
[nitpick] I understand you have to make these checks now because `ep.processKey` is no longer a pointer, but the actual value. But this is can be a problem to maintain in the future in case this is used elsewhere and the same check has to be replicated or if the number of properties in `processInfo` changes. 

I have couple of suggestions:
1) Extract this check to a `func (p processInfo) empty() bool` receiver function.
2) Check only a single property. It seems like that it's either all set or none. Not sure if this is always true though
3) Create an `emptyProcessInfo` var definition that can be compared to. If more properties are added you don't need to update the check. Like:
```go
var (
	emptyProcessInfo = processInfo{}
)

func someFn() {
	// ...
	p := getProcessKey(nil)
	if p == emptyProcessInfo {
		// ...
	}
}
```

Reply:
I went with option number 3.